docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/creating-vehicle-description.md

```diff
@@ -0,0 +1,205 @@
+    /**:
+      ros__parameters:
+        wheel_radius: 0.27855 # The radius of the wheel, primarily used for dead reckoning.
+        wheel_width: 0.155 # The lateral width of a wheel tire, primarily used for dead reckoning.
+        wheel_base: 1.60 # between front wheel center and rear wheel center
+        wheel_tread: 0.959 # between left wheel center and right wheel center
+        front_overhang: 0.245 # between front wheel center and vehicle front
+        rear_overhang: 0.245 # between rear wheel center and vehicle rear
+        left_overhang: 0.075 # between left wheel center and vehicle left
+        right_overhang: 0.075 # between right wheel center and vehicle right
+        vehicle_height: 2.
+        max_steer_angle: 0.70 # [rad]
+    ```
+
+### 4. vehicle.fbx
+
+`vehicle.fbx` is a 3D model file used for visualization in RVIZ.
+You must add the fbx or dae file of your own vehicle.
+
+The 3D model of Lexus vehicle prepared for sample_vehicle_launch is [here](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main/sample_vehicle_description/mesh).
+
+### 5. vehicle.xacro
+
+`vehicle.xacro` is a macro file that defines the vehicle model.
+
+The original [vehicle.xacro](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_description/urdf/vehicle.xacro) file for sample_vehicle_launch is given below:
+
+??? note "vehicle.xacro for sample_vehicle_launch"
+
+    ```yaml
+    <?xml version="1.0"?>
+    <robot xmlns:xacro="http://ros.org/wiki/xacro">
+      <!-- load parameter -->
+      <xacro:property name="vehicle_info" value="${xacro.load_yaml('$(find sample_vehicle_description)/config/vehicle_info.param.yaml')}"/>
+
+      <!-- vehicle body -->
+      <link name="base_link">
+        <visual>
+          <origin xyz="${vehicle_info['/**']['ros__parameters']['wheel_base']/2.0} 0 0" rpy="${pi/2.0} 0 ${pi}"/>
+          <geometry>
+            <mesh filename="package://sample_vehicle_description/mesh/lexus.dae" scale="1 1 1"/>
+          </geometry>
+        </visual>
+      </link>
+    </robot>
+    ```
+
+## 2. VEHICLE-ID_vehicle_launch
+
+`vehicle_interface.launch.xml` is the launch file related to the drive system. Please modify it according
+to the configuration of your vehicle's drive system.
+
+### Folder Structure
+
+The folder structure of forked `VEHICLE-ID_vehicle_launch` should be like this:
+
+```diff
+VEHICLE-ID_vehicle_launch/
+   └─ launch/
+         └─ vehicle_interface.launch.xml
+```
+
+If you are operating multiple vehicles, use the vehicle_id to switch to the corresponding configuration for each vehicle.
+
+The default file is [vehicle_interface.launch.xml](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_launch/launch/vehicle_interface.launch.xml) for sample_vehicle_launch is given below:
+
+```yaml
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+<arg name="vehicle_id" default="$(env VEHICLE_ID default)"/>
+</launch>
+```
```

Comment:
Maybe we can add suggestion for user, they can add their own vehicle_interface launch here.